### ISSUE TITLE
[storage] anvil-manage-server-storage CLI options must be cleaned up

### ISSUE THREAD

**Issue by fabbione** [ClusterLabs/anvil#654]
The CLI usage is confusing at best and it needs to be reworked to be easy to understand and consistent.

- [ ] drop the concept of --optical and --disk and rename both to --device. anvil already knows what type of device it is and how to act on it.
- [ ] do NOT attempt to map internal VM disk names with outside. For example:```Optical Drives:
- Target: [sda], boot: [02], ISO: [/mnt/shared/files/CentOS-Stream-9-latest-x86_64-dvd1.iso]
- Target: [sdb], boot: [03], ISO: [/mnt/shared/files/deploy1.iso]``` will never be correct, as internally they are seen as sr0 / sr1. Offer instead a simple sequence number: disk0 / disk1 / cdrom0 / cdrom1.. etc.
- [ ] convert options to actions. For example --add --grow --remove (that is not there), should become: "add --size XGB" (to add a new disk) "grow --size XGB" (to grow to absolute XGB) or "grow --size +XGB" (to grow XGB more, error out on -XGB as shrinking is not supported)
- [ ] --storage-group should be optional if there is only one storage group.

**Comment by digimer** [comment]
These are good enhancements, for sure, but also this can wait until 3.2. Exceedingly few people will ever use the back-end to manage storage. The priority for 3.1 is UI features/enhancements.

**Comment by fabbione** [comment]
just keep in mind we need the cli to run tests in CI

**Comment by digimer** [comment]
Yup. For now, it works, if clunky. When I clean up the switches, I'll coordinate to get the tests updated as well

**Comment by digimer** [comment]
This is listed as `man page` but it requires some support in the tool to align the switches to those requested.

**Comment by digimer** [comment]
Coordinate switch changes with @ylei-tsubame
